explorer/testdata/namespace/namespace_alias.carbon

```diff
@@ -5,16 +5,24 @@
 // AUTOUPDATE
 // RUN: %{not} %{explorer-run}
 // RUN: %{not} %{explorer-run-trace}
+// CHECK:STDOUT: result: 0
 
 package ExplorerTest api;
 
 namespace N;
+namespace N.M;
 fn N.F() {}
-// TODO: This should work.
-// CHECK:STDERR: COMPILATION ERROR: {{.*}}/explorer/testdata/namespace/fail_namespace_alias.carbon:[[@LINE+1]]: could not find `namespace N;`
-alias M = N;
+fn N.M.F() {}
```

Comment:
This should probably use a different name than `F` so that it's more obvious if there's a lookup issue.